### PR TITLE
Redo SMPP docs

### PR DESCRIPTION
**Pull request by jwldk**

### CODE REVIEW

**Review by necrophcodr — DISMISSED** [review]
It might be useful to elaborate on how to use the smpp connections, but I guess the customers using these will already be aware.

File: docs/smpp.rst [hunk 1]
```diff
@@ -5,14 +5,15 @@ We offer SMPP connection for select customers. Contact sales@gatewayapi.com to g
 
 Connection
 ----------
-Use the following to connect
+Use the following to connect. We recommend to connect to two hosts. When we do scheduled maintainence, only one host is restarted at a time. By keeping a connection to two hosts, constant
+connectivity can be achieved.
```
Comment (cknv) [inline docs/smpp.rst]: Thats a weird line break ? I guess rst don't care about that ?

File: docs/smpp.rst [hunk 2]
```diff
@@ -39,20 +40,53 @@ enquire_link            0x00000015
...
+
+We support the following common TLV fields for `submit_sm` as well as one our own custom for use with message classes.
+
+======== ======================= ======== ========================
+Tag      Name                    Size     Description
+======== ======================= ======== ========================
+0x0005   dest_addr_subunit       1 byte   For "flash" sms
+0x0204   user_message_reference  2 bytes  User assigned reference for delivery reports
+0x0424   message_payload         1 byte   For sending messages longer than 255 octets
+0x2900   message_class           octet    To send with a specific GatewayAPI messageclass
```
Comment (cknv) [inline docs/smpp.rst]: Just one octet ? Unless I am missing something I think it's a bunch of bytes that happens to be a UTF8 encoded string matching our message classes

**Review by necrophcodr — DISMISSED** [review]

**Comment by cknv** [comment]
Did we want to keep the old and the new SMPP docs side by side ?

File: docs/smpp.rst [hunk 3]
```diff
@@ -84,11 +83,11 @@ TLV fields
...
-0x0424   message_payload         1 byte   For sending messages longer than 255 octets
-0x2900   message_class           octet    To send with a specific GatewayAPI messageclass
-======== ======================= ======== ========================
+======== ======================= ============= ========================
+Tag      Name                    Size          Description
+======== ======================= ============= ========================
+0x0005   dest_addr_subunit       1 byte        For "flash" sms
+0x0204   user_message_reference  2 bytes       User assigned reference for delivery reports
+0x0424   message_payload         1 byte        For sending messages longer than 255 octets
+0x2900   message_class           octetstring   To send with a specific GatewayAPI messageclass
```
Comment (necrophcodr) [inline docs/smpp.rst:92]: Do we have a length at all?

Comment (jwldk) [reply docs/smpp.rst:92]: I dont think we a length here. Just variable string.

**Review by necrophcodr — APPROVED** [review]

**Pull request merged**